lib/core/src/persist/chain.rs

```diff
@@ -69,6 +67,7 @@ impl Persister {
                 accept_zero_conf = :accept_zero_conf,
                 server_lockup_tx_id = :server_lockup_tx_id,
                 user_lockup_tx_id = :user_lockup_tx_id,
+                claim_address = :claim_address,
```

Comment:
Is this because the claim address is now synced for "receive onchain"? If so don't we need also to add it to the sync merge?

Reply:
@roeierez this is because we recover it from chain data, so the swap needs updating. We did agree that it doesn't need to be synced because of this

Reply:
Got it.